jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/mgt/Backup.java

```diff
@@ -61,18 +61,15 @@ public static String chooseFileName(String dsName) {
     // same dataset multiple times at the same time.
     private static Set<DatasetGraph> activeBackups = Collections.newSetFromMap(new ConcurrentHashMap<>());
 
-    /** Perform a backup.
-     *  A backup is a dump of the dataset in compressed N-Quads, done inside a transaction.
+    /**
+     * Perform a backup.
+     * <p>
+     * A backup is a dump of the dataset in compressed N-Quads, done inside a transaction.
      */
     public static void backup(Transactional transactional, DatasetGraph dsg, String backupfile) {
         if ( transactional == null )
             transactional = new TransactionalNull();
-        transactional.begin(ReadWrite.READ);
-        try {
-            Backup.backup(dsg, backupfile);
-        } finally {
-            transactional.end();
-        }
+        Txn.executeRead(transactional,()->backup(dsg, backupfile));
```

Comment:
missing space after comma (just in case it might be picked later by linter/IDE/etc?)

Reply:
Fixed!